index_configuration_tool/main.py

```diff
@@ -128,9 +146,9 @@ def print_report(index_differences: tuple[set, set, set]):  # pragma no cover
     print("Indices to create: " + utils.string_from_set(index_differences[0]))
 
 
-def run(config_file_path: str) -> None:
+def run(args: argparse.Namespace) -> None:
     # Parse and validate pipelines YAML file
-    with open(config_file_path, 'r') as pipeline_file:
+    with open(args.config_file_path, 'r') as pipeline_file:
```

Comment:
worth checking that this value exists/throwing an intentional exception here? Or does argparse handle it for you?

Reply:
`config_file_path` is a positional argument, so `argparse` will ensure that it is present